tasks/deploy/fetch.js

```diff
@@ -95,8 +96,10 @@ module.exports = function (gruntOrShipit) {
     function fetch() {
       var fetchCommand = 'git fetch' +
         (shipit.config.shallowClone ? ' --depth=1 ' : ' ') +
-        'shipit -p --tags';
+        'shipit -p --tags' + 
+	(shipit.config.updateSubmodules ? ' --recurse-submodules' : '');
 
+      shipit.log('fetch submodules: ' + shipit.config.recursiveFetch);
```

Comment:
Ahh sorry this comment is misleading - I will remove and add a comment to the updateSubmodules function.  This just ensures that the submodule information is available to shipit when the update is done later.